chpl/chpl-resources/src/main/resources/email.properties

```diff
@@ -354,10 +354,10 @@ developer.missingAttestationChangeRequest.report.salutation=Hello %s,
 developer.missingAttestationChangeRequest.report.paragraph=Emails were sent to the following developers:
 
 #URL Uptime Report
-urlUptime.report.filename=URL Uptime
-urlUptime.report.subject=CHPL URL Uptime Report
-urlUptime.report.heading=CHPL URL Uptime Report
-urlUptime.report.paragraph1.body=See attached file
+serviceBasedUrlUptime.report.filename=Service Based URL Uptime
```

Comment:
I see the text all around "Service Based URL" but in the UI and in the data model we call it "Service Base URL List". I just want to make sure our terminology is how we want it. Should it be the past-tense Based vs Base? I thought it was like the "base" URL ("Base" as in the foundational URL for all Services) although I truly do not know what is correct.

Reply:
![image](https://github.com/chpladmin/chpl-api/assets/24817202/efb2b11b-86eb-4bc7-9fc4-b14f52bce5bb)
I based my naming on the ticket description and the wording that I hear Jim using.